core/common/src/main/java/alluxio/Configuration.java

```diff
@@ -200,11 +209,11 @@ public static void set(String key, String value) {
    * @return the value for the given key
    */
   public static String get(String key) {
-    if (!PROPERTIES.containsKey(key)) {
+    String raw = PROPERTIES_MAP.get(key);
```

Comment:
It was called `raw` before this pr. But I'm changing this variable in all related function to rawValue.